### PR TITLE
[SYTO-822] Create and Style The Latest Component

### PR DESCRIPTION
**Pull request by pgurley**
https://jira.natgeo.com/browse/SYTO-822

This story creates "The Latest" component, which is rendered by a component-data modeled (https://jira.natgeo.com/browse/AEM-5800) as "ContentPackage". The "ContentPackage" component will also contain the "Most Read" component. That work is slated for a future sprint. I imagine during the work for https://jira.natgeo.com/browse/SYTO-885 and https://jira.natgeo.com/browse/SYTO-886 there will be some refactoring of the instantiation of these components.

It should also be noted that the styling of the individual story cards is being handled in different stories as well.

To test:

- Pull feature/SYTO-822
- Run `npm test` to verify no errors
- Run `gulp serve`
- Load http://localhost:3000/contentPackages/contentPackage/
- Verify 4 cards are rendered
- Verify heading value can be updated
- Don't worry about layout of story cards yet

### CODE REVIEW

File: app/contentPackages/contentpackage/_styles.scss [hunk 1]
```diff
@@ -0,0 +1,73 @@
+$mt3_black: #000;
+$mt3_white: #fff;
+$mt3_yellow: #ffcc00;
+$mt3_dark: #222222;
```
Comment (benjachapman) [inline app/contentPackages/contentpackage/_styles.scss]: Are there actual Mortar color variables that could be used here? I think there is at least for "black"

File: app/contentPackages/contentpackage/scripts/components/FourUp.jsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+    let currentCard = card;
+    switch(index) {
+      case 1:
+        currentCard.config.aspectRatio = "16:9";
+      break;
+      case 0:
+        currentCard.config.aspectRatio = "3:2";
+      break;
+      default:
```
Comment (benjachapman) [inline app/contentPackages/contentpackage/scripts/components/FourUp.jsx]: ought this "default" statement be removed? not sure if having a blank default case has some value i'm unaware of

File: app/contentPackages/contentpackage/_styles.scss (hunk 1 above)
Comment (pmourelle) [inline app/contentPackages/contentpackage/_styles.scss]: Actually, all of them :)
`/app/styles/themes/default/vars/_colors.scss`
`/app/styles/global/vars/_theme.scss`

File: app/contentPackages/contentpackage/scripts/ContentPackagePestle.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+import React from 'react';
+import * as ReactDOM from 'react-dom';
```
Comment (pmourelle) [inline app/contentPackages/contentpackage/scripts/ContentPackagePestle.js]: Is that `*` needed?

**Comment by benjachapman** [comment]
Only a small nit or two to pick, looks great. tests & configurator working well.

The only concern i have here is the naming of the component, but I think we can address that on the go-forward. We'll need to discuss that with the larger group i reckon.

File: test/app/contentPackages/contentPackage/components/fourUp_test.js [hunk 4]
```diff
@@ -0,0 +1,491 @@
+'use strict';
+
+import FourUpComponent from '../../../../../app/contentPackages/contentpackage/scripts/components/FourUp.jsx';
+import {shallow} from 'enzyme';
+import React from 'react';
+
+describe('FourUp Component', () => {
+  let el;
+  const options = {
+      'id': '3t35sdfj',
```
Comment (pmourelle) [inline test/app/contentPackages/contentPackage/components/fourUp_test.js]: Would be much easier to read if all json lives in a separate file.

File: app/contentPackages/contentpackage/_styles.scss (hunk 1 above)
Comment (pgurley) [inline app/contentPackages/contentpackage/_styles.scss]: You caught my TODO I forgot about! Making this update

File: app/contentPackages/contentpackage/scripts/components/FourUp.jsx (hunk 2 above)
Comment (pgurley) [inline app/contentPackages/contentpackage/scripts/components/FourUp.jsx]: I propose adding a comment to the default value explaining what's going on vs. leaving it out altogether

Something like:
```
      default:
        // Default will be whatever was passed in as a prop. We're only overriding the first two cards
```

Thoughts?

File: app/contentPackages/contentpackage/scripts/ContentPackagePestle.js (hunk 3 above)
Comment (pgurley) [inline app/contentPackages/contentpackage/scripts/ContentPackagePestle.js]: I don't believe so. Removing it

File: app/contentPackages/contentpackage/scripts/components/FourUp.jsx (hunk 2 above)
Comment (benjachapman) [inline app/contentPackages/contentpackage/scripts/components/FourUp.jsx]: ah, yeah fine by me

File: test/app/contentPackages/contentPackage/components/fourUp_test.js (hunk 4 above)
Comment (pgurley) [inline test/app/contentPackages/contentPackage/components/fourUp_test.js]: OK, I broke this out into a separate file

**Comment by pgurley** [comment]
All feedback has been addressed and this is ready for another round

**Comment by benjachapman** [comment]
Looks great, thanks Phil. merging.

**Comment by pmourelle** [comment]
NOOICE!

**Pull request merged**